.github/workflows/main.yml

```diff
@@ -22,3 +22,69 @@ jobs:
               uses: docker://oskarstark/php-cs-fixer-ga
               with:
                 args: --dry-run --diff-format udiff
+
+    phpunit:
+        name: PHPUnit
+        runs-on: ubuntu-latest
+        env:
+            STABILITY: ''
+            PHPUNIT_FLAGS: -v --testsuite main
+            SYMFONY_VERSION: ''
+            SYMFONY_DEPRECATIONS_HELPER: max[self]=2
+            COVERAGE: false
+        strategy:
+            matrix:
+                include:
+                    - PHP_VERSION: 7.2
+                      COMPOSER_FLAGS: --prefer-stable --prefer-lowest
+                      SYMFONY_PHPUNIT_VERSION: 7.5
+
+                    - PHP_VERSION: 7.2
+                    - PHP_VERSION: 7.3
+                    - PHP_VERSION: 7.4
+                      COVERAGE: true
+
+                    - PHP_VERSION: 7.2
+                      PHPUNIT_FLAGS: -v --testsuite doctrine
+                      COMPOSER_FLAGS: --prefer-source
+                      SYMFONY_VERSION: 3.4.*
+                      SYMFONY_PHPUNIT_VERSION: 7.5
+
+                    - PHP_VERSION: 7.4
+                      SYMFONY_VERSION: 3.4.*
+                    - PHP_VERSION: 7.2
+                      SYMFONY_VERSION: 4.4.*
+                    - PHP_VERSION: 7.4
+                      SYMFONY_VERSION: 5.1.*
+
+                    - PHP_VERSION: 7.4
+                      STABILITY: dev
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2.0.0
+            - name: "Setup PHP"
+              uses: shivammathur/setup-php@2.4.2
+              with:
+                coverage: none
+                php-version: ${{ matrix.PHP_VERSION }}
+            - name: "Configure composer minimum stability"
+              if: matrix.STABILITY == 'dev'
+              run: composer config minimum-stability ${{ matrix.STABILITY }}
+            - name: "Install dependencies with composer"
+              if: matrix.SYMFONY_VERSION == ''
+              run: composer update ${{ matrix.COMPOSER_FLAGS }} --prefer-dist --no-interaction
+            - name: "Install dependencies with composer"
+              if: matrix.SYMFONY_VERSION != ''
+              run: |
+                composer require "symfony/symfony:${{ matrix.SYMFONY_VERSION }}" --no-update
```

Comment:
You may use flex instead. 

In the shivammathur/setup-php, specify "tools: flex" and then use `SYMFONY_REQUIRE` instead of `SYMFONY_VERSION `